source/tyr/tyr/default_settings.py

```diff
@@ -159,3 +159,5 @@
 # https://flask-sqlalchemy.palletsprojects.com/en/2.x/signals/
 # deprecated and slow
 SQLALCHEMY_TRACK_MODIFICATIONS = os.getenv('TYR_SQLALCHEMY_TRACK_MODIFICATIONS', False)
+
+POST_DATA_TO_TYR = os.getenv('TYR_POST_DATA_TO_TYR', None)
```

Comment:
```suggestion
# Url of a secondary Tyr. The data posted to an instance of this Tyr via the '/jobs' endpoint will be reposted to this url
POST_DATA_TO_TYR = os.getenv('TYR_POST_DATA_TO_TYR', None)
```

Reply:
I would also be closer to what it really does (it could be something else than tyr):
```suggestion
REPOST_DATA_TO_URL = os.getenv('TYR_REPOST_DATA_TO_URL', None)
```

Reply:
I would prefer : REPOST_DATA_TO_TYR